quiz.rb

```diff
@@ -2,19 +2,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  def name
+    @name
+  end
+
+  def age
+    @age
+  end
+
+  def breed
+    @breed
+  end
+end
+
 # Question 2
 # Create a new Dog object.
 
+bundleOfAwesome = Dog.new("Emily", "Boxer - German Shephard", 6)
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+  def initialize(name, age, location)
+    @name = name
+    @age = age
+    @location = location
+  end
+
+  attr_reader :age
```

Comment:
This is a bit of a minor thing, but it's conventional to put `attr_reader` and `attr_accessor` before the `initialize` method.